src/progress/stories/CircularProgress.stories.tsx

```diff
@@ -9,7 +9,7 @@ import {
 } from "./storybook-progress-utils";
 
 export default {
-  title: "Component/Progress/Circular",
+  title: "Progress Circular",
```

Comment:
I think we can keep them colocate. it will be easier to browse them and make the tree smaller overall.

Reply:
Okay Progress/Circular.